From: Thomas Haller <thaller@redhat.com>
Date: Mon, 18 Jun 2018 11:24:00 +0200
Subject: tests: don't exit test-networkmanager-service.py after 20 seconds

Tests might just take longer than 20 seconds.

Also, we already watch stdin to determine whether the service
should exit.

(cherry picked from commit 4e18ef49bf2527b4c29a0bdff2faddf0194dd5bd)
(cherry picked from commit 296da44235815ff32202f2afe9c990e94e4824ea)
---
 tools/test-networkmanager-service.py | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/tools/test-networkmanager-service.py b/tools/test-networkmanager-service.py
index f6da6e1..7fbed7e 100755
--- a/tools/test-networkmanager-service.py
+++ b/tools/test-networkmanager-service.py
@@ -2252,14 +2252,9 @@ def main():
     id1 = GLib.IOChannel(0).add_watch(GLib.IOCondition.HUP,
                                       lambda io, condition: gl.mainloop.quit() or True)
 
-    # also quit after inactivity to ensure we don't stick around if the above fails somehow
-    id2 = GLib.timeout_add_seconds(20,
-                                   lambda: gl.mainloop.quit() or True)
-
     gl.mainloop.run()
 
     GLib.source_remove(id1)
-    GLib.source_remove(id2)
 
     gl.agent_manager.remove_from_connection()
     gl.dns_manager.unexport()
